astropy/coordinates/distances.py

```diff
@@ -75,6 +76,8 @@ class Distance(u.SpecificTypeQuantity):
         If ``z`` is provided with a ``unit`` or ``cosmology`` is provided
         when ``z`` is *not* given, or ``value`` is given as well as ``z``.
 
+        If none of ``value``, ``z``, ``distmod``, or ``parallax`` were given
```

Comment:
```suggestion
        If none of ``value``, ``z``, ``distmod``, or ``parallax`` were given.
```

Reply:
Darn. missed this in the 153 files of changes. Going to force push this in.